### PR TITLE
Added Build badge for travis CI to README

### LINKED ISSUE #108: Doc: Add Build badge for Travis CI to README
## Description
As a contributor,
I need to see Build badge,
so that I can see the status of the latest Travis CI build.

## Mocks
NA

### Update [Required]
- [ ] Add Build badge for Travis CI build. [Refer to the following resource on how to do it](https://docs.travis-ci.com/user/status-images/)

### Enhamcement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by AltCtrlDel1999**
### Description

Added Build status badge into the README file

Fixes #ISSUE
#108 
### Type of Change:

- Documentation

### How Has This Been Tested?
Checked Manually

### CODE REVIEW

**Comment by AltCtrlDel1999** [comment]
@mtreacy002 Please review this PR

File: src/config.js [hunk 1]
```diff
@@ -1 +1,2 @@
 export const BASE_API = process.env.REACT_APP_BASE_API ?  process.env.REACT_APP_BASE_API : "http://localhost:5000"
+//export const BASE_API = process.env.REACT_APP_BASE_API ?  process.env.REACT_APP_BASE_API : "bridgeintech-bit-heroku-psql.herokuapp.com"
```
Comment (mtreacy002) [inline src/config.js]: @AltCtrlDel1999 , please remove this commented line.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@AltCtrlDel1999 , please check my feedback below.

**Comment by AltCtrlDel1999** [comment]
@mtreacy002 Done with changes.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@AltCtrlDel1999 . There is a conflict caused by the recent merge to upstream develop. Can you please integrate this into your code by pull --rebase from upstream develop? Thanks

**Comment by AltCtrlDel1999** [comment]
@mtreacy002 Done

**Review by mtreacy002 — APPROVED** [review]
LGTM 🎉🎉 . Thank you for your contribution, @AltCtrlDel1999

**Pull request merged**